packages/eslint-plugin/src/rules/ban-types.ts

```diff
@@ -107,9 +107,14 @@ export default createRule<Options, MessageIds>({
     type: 'suggestion',
     docs: {
       description: 'Disallow certain types',
-      recommended: 'recommended',
     },
     fixable: 'code',
+    deprecated: true,
```

Comment:
As above - why is it deprecated? It is still a useful utility rule even if we've emptied the default config.